scalafmt-tests/src/test/resources/default/Select.stat

```diff
@@ -62,7 +62,7 @@ val lastToken = owner.body.tokens.filter {
   case _: Whitespace | _: Comment => false
   case _ => true
 } // edge case, if body is empty expire on arrow.
-.lastOption.getOrElse(arrow)
+  .lastOption.getOrElse(arrow)
```

Comment:
IMHO previous behavior was prettier 🤔 
@tanishiking @olafurpg what do you think?

Reply:
I think it's acceptable to add an indent here 👍 , and it's more consistent if scalafmt adds an indent before methods for any objects (even if it's Block).

FYI [prettier add an indent in this case](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEA3TBeTYAHSk0wA8lMBycu6gGhLIE8rqXPGSBfTAen6Y0ACwgBXADYATTAHc4mAIbTZS0gEso0hFmiYYIxQDMIkyRDlaA5pkla4AfhIA6cgAoAlCAYgIABxgNdGRQJQAncMsABQiENGQQJWwIDWkfEAAjcKUwAGs4GABlf1ybZBhw8ThfERgAW0kAdRENeDRSsDgi+LaNbDaWRLA0BN8tNDhwmGic63qlZGMlSUnfACs0cgAhHPzCoqV6uAAZByWVtZBN8iKbSTgARXEIeAvVmpBS8MnwxMylJk4JIMv5wloYE00oZkAAOAAMvjBEEmTRy-kSYLgv2wcAyAEcXvBZgEEiglGgALRQOBwHTpXzhOCEjRM2ZKeaLJDLD6+Sb1DQVKqfND3J5EvHcy6fGCAqHSGFIABMvkqSg09ig1gAwhB6gtElBoHjfOJJgAVQFknlXbDVACS2l0RTA4MCAEFtEUYCwHu9JjweEA)

Reply:
I like the new behavior.